packages/ra-core/src/dataProvider/useUpdate.ts

```diff
@@ -431,7 +431,7 @@ export type UseUpdateOptions<
     RecordType,
     MutationError,
     Partial<UseUpdateMutateParams<RecordType>>
-> & { mutationMode?: MutationMode };
+> & { mutationMode?: MutationMode; returnPromise?: boolean };
```

Comment:
please test and document that option

Reply:
It is used in the `useEditController` in pessimistic mode, so it is tested. I'm not sure we really need to document this option as it's pretty advanced, and the option was not documented in the `updateOptions` type in the first place.